describealign.py

```diff
@@ -182,7 +198,7 @@ def tokenize_audio_dither(media_arr, slow_timings):
   # this is the continued fraction [0;1,N-2,1,1,1,...], where the trailing ones give phi
   fast_rate = 1. / (1 + 1. / (DITHER_PERIOD_STEPS - 2 + (np.sqrt(5) + 1) / 2.))
   fast_spec, fast_timings = tokenize_audio(media_arr, fast_rate)
-  
+

```

Comment:
Don't mind the trailing white spaces being deleted, but I kind of like the empty lines having matching tabbing as it's less fiddly when editing.  Is there a reason to get rid of it?

Reply:
It's generally considered bad form as it adds extra characters to the file that have little use, as most editors will auto-indent when you add a new line in a code block.

Reply:
Found this discussion in which the top answer says PEP8 doesn't specify and both are legal, while the close second answer says it should be added as it's important for readability when display whitespace is enabled, especially with two-space indentation like we have here.
https://stackoverflow.com/questions/2727988/python-indentation-in-empty-lines

Reply:
This change was automatically implemented by VSCode, and reading through the code it seems to already be inconsistent. Is this a linter check we want to implement?

Reply:
That SO answer also mentions that the PEP8 doesn't take a stance on it, but under [this section](https://peps.python.org/pep-0008/#other-recommendations) it does specifically mention removing all trailing whitespace, not just whitespace after statements.

Reply:
Trailing whitespace is whitespace after statements.  I'd prefer to keep the non-trailing whitespace, but if it would bother you I also don't really mind that much either way.  Your call here.

Reply:
For now I have reverted these changes and added a vscode config file to disable removing trailing whitespace. As for the future, I think it would be beneficial to define a lint/formatter config either way, to enforce consistency.

As for removing the trailing whitespace on blank lines, `git diff --check` will yell at you about it by default, and most editors will automatically trim ALL trailing whitespace (including blank lines). Git specifically refers to both lines containing only whitespace and lines containing code followed by whitespace as 'lines with trailing whitespace', and thus default checks in the diff will alert on both. I think for this reason, we should migrate to stripping all trailing whitespace, to be consistent with the defaults in VSCode, Git, PyCharm, the `black` formatter, `ruff`, etc.